pkgs/development/libraries/libtickit/default.nix

```diff
@@ -1,54 +1,67 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, nix-update-script
 , pkg-config
 , libtool
 , perl
 , libtermkey
 , unibilium
 }:
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libtickit";
   version = "0.4.3";
 
   src = fetchFromGitHub {
     owner = "leonerd";
     repo = "libtickit";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
```

Comment:
Since this a FOD and will not just work when changing version, I see no point in converting this line alone

Reply:
I went back to the use of the `rec` keyword for the `version` variable. However, could you elaborate on this point: How come it will not work because this is a fixed output derivation?